sensu/plugins/check-controller-tuning.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
+##
+## Check dedicated controller ethernet tuning
+##
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-plugin/check/cli'
+
+class CheckControllerTuning < Sensu::Plugin::Check::CLI
+  def run
+    tuning_status = `service ethernet-tuning status`
```

Comment:
Thinking about error handling...
Are we assuming that this check will only be running on dedicated controllers ?
I think that is an ok assumption.
But if not we should have some error handling in case this check get's run on a host that does not have ethernet-tuning service installed.

Reply:
could you not use https://github.com/blueboxgroup/ursula-monitoring/blob/master/sensu/plugins/check-procs.rb and check for the actual process to be running ?

If you do need an actual service status, remember that you may be dealing with multiple init systems ( systemd vs sysinit, vs upstart ) so test appropriately and abstract the service name into an argument so that you can reuse the check for any service.

Reply:
@paulczar Thanks for the feedback. So in this instance, the init script that is installed simply utilizes ethtool to set/check settings. `service ethernet-tuning status` appears to only run ethtool to see if the settings are present, so I have included that same check in a commit which should should work on any init system.